packages/wallets/src/components/WalletsCarousel/WalletsCarousel.tsx

```diff
@@ -1,24 +1,56 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { useEventListener } from 'usehooks-ts';
 import { useActiveWalletAccount } from '@deriv/api-v2';
 import { AccountsList } from '../AccountsList';
 import { WalletsCarouselContent } from '../WalletsCarouselContent';
 import { WalletsCarouselHeader } from '../WalletsCarouselHeader';
+import './WalletsCarousel.scss';
 
 const WalletsCarousel: React.FC = () => {
     const [isWalletSettled, setIsWalletSettled] = useState(true);
+    const [isContentScrolled, setIsContentScrolled] = useState(false);
+    const [heightFromTop, setHeightFromTop] = useState(0);
     const { data: activeWallet, isLoading: isActiveWalletLoading } = useActiveWalletAccount();
 
+    const containerRef = useRef<HTMLDivElement>(null);
+
+    const handleScroll = useCallback(() => {
```

Comment:
woudl be really cool to add comments to understand whats the purpose of this bit (or add tests and make the tests act as a feature documentation)

Reply:
Added comments 👍 Also renamed isContentScrolled to avoid confusion